src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,46 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Block {
+	public static final String EXAMPLE = "12345";
+	public static final String MESSAGE_BLOCK_CONSTRAINTS = "Block should be in number";
+	public static final String BLOCK_VALIDATION_REGEX = "\\d+";
+	public final String value;
+
+	 /**
+     * Validates given block.
+     *
+     * @throws IllegalValueException if given block string is invalid.
+     */
+	public Block(String block) throws IllegalValueException{
+		 if (!isValidBlock(block)) {
+	            throw new IllegalValueException(MESSAGE_BLOCK_CONSTRAINTS);
+	        }
+		this.value = block;
+	}
+	
+	 /**
+     * Returns true if a given string is a valid address block.
+     */
+	public static boolean isValidBlock(String test) {
+        return test.matches(BLOCK_VALIDATION_REGEX);
+    }
+
+	@Override
+	public String toString() {
```

Comment:
This method acts as your getter. It would be even better to have an actual getter.